CODEOWNERS

```diff
@@ -18,3 +18,73 @@ audit_exceptions/permitted_formula_license_mismatches.json              @Homebre
 audit_exceptions/provided_by_macos_depends_on_allowlist.json            @Homebrew/tsc
 audit_exceptions/universal_binary_allowlist.json                        @carlocab @Homebrew/tsc
 audit_exceptions/versioned_formula_dependent_conflicts_allowlist.json   @Homebrew/tsc
+
+# TODO: Remove when https://github.com/Homebrew/homebrew-core/pull/98809 is merged.
+Formula/apngasm.rb             @carlocab @Homebrew/tsc
```

Comment:
If there are objections from @Homebrew/tsc for all the extra notifications this will give you, we can just put @SMillerDev here 😄

Reply:
At least for me: I don't mind either way. I'm fine with it being just you even but if you want a second person than that's also fine.

Reply:
I think we've moved to having more than one code owner whenever a file has one since we started requiring code owner approvals for PRs. (Code owner approval was previously not required.)

Of course, @BrewTestBot currently doesn't care about `CODEOWNERS`, but I imagine that's something we want to change at some point.

Reply:
Or, I guess, alternatively, we could've made @BrewTestBot the code owner if the goal was just to prevent those PRs from getting merged? Not sure how `brew pr-automerge` treats approved-but-not-code-owner-approved PRs though.